centreon/www/front_src/src/Resources/Details/tabs/Graph/ChartGraph.tsx

```diff
@@ -89,7 +89,7 @@ const ChartGraph = ({
   return (
     <>
       <FederatedComponent
-        path="/anomaly-detection/enableThresholdLines"
+        path="/anomaly-detection/EnableThresholdLines"
```

Comment:
i find that wierd since the path declared in moduleFederation.json is /anomaly-detection/enableThresholdLines , also i didn t get this crash on my environement

Reply:
Indeed, that's weird are you sure that Anomaly Detection were on the right according according to Centreon web ?